### ISSUE TITLE
Typecast shows (Request textDocument/documentHighlight failed)

### ISSUE THREAD

**Issue by Eveheeero** [ktnrg45/vs-code-cython#3]
## Comment block commonly cause stuck in definition block

I was trying to comment cdef class I defined, however when I typed """ at incorrect indent, all code from the next line is highlighted in red.

The red highlight didn't disappear even when the comment was completed, and the highlight disappeared when the file was turned off.

It seems think that error occurred cause I didn't match indent of comment block, but  it's a problem that the error need to close and open the file to correct it.

I have a lot of comment block with right indent, and when I was trying to make new comment block, same error occurs again with other error code (origin error code : "Expected indented blockCython", now error code : "String literal is unterminated") and stuck also.

**Comment by ktnrg45** [comment]
Thanks for reporting this. Can you please post an example of this?

**Comment by Eveheeero** [comment]
After that, I figured out that the comment block wasn't the problem. The problem was type casting of <long> or <long long> (also seen with unsigned keyword).

Writing that typecast shows (Request textDocument/documentHighlight failed), and if any highlighting occurs in that state, highlighting doesn't end.

```python
def test() -> str:
    """
    """
    t = 0
    e = <long>t
    return e
```
If you delete one of the """ in the code above and retype it again, you can see that the highlighting won't end.

**Comment by ktnrg45** [comment]
Fixed the error with casting. Should be fixed in 0.1.4.
